src/Symfony/Component/VarDumper/Tests/Caster/ReflectionCasterTest.php

```diff
@@ -550,7 +554,7 @@ public function testGenerator()
         Symfony\Component\VarDumper\Tests\Fixtures\GeneratorDemo::foo()
         ›     yield 1;
         › }
-        › 
+        ›
```

Comment:
@nicolas-grekas The fabot reported this: https://fabbot.io/report/symfony/symfony/56488/ddf9e3b7b426612f6edc37c4cf41c00f754ad4de

I'm not sure if I should edit it in my PR to comply or let the status red.